phpcs.xml.dist

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="WordPress Coding Standards" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/squizlabs/PHP_CodeSniffer/master/phpcs.xsd">
+
+  <!-- See https://github.com/squizlabs/PHP_CodeSniffer/wiki/Annotated-ruleset.xml -->
+  <!-- See https://github.com/WordPress/WordPress-Coding-Standards/blob/develop/.phpcs.xml.dist -->
+
+  <!-- Set a description for this ruleset. -->
+  <description>Coding standards used for checking the Twenty Twenty theme.</description>
+
+  <!-- Exclude patterns -->
+  <exclude-pattern>/vendor/*</exclude-pattern>
+  <exclude-pattern>/node_modules/*</exclude-pattern>
+
+  <!-- Include the WordPress ruleset, with space for exclusions if necessary. -->
+  <rule ref="WordPress-Extra">
+    <exclude name="Generic.WhiteSpace.ScopeIndent.Incorrect" />
+    <exclude name="Generic.WhiteSpace.ScopeIndent.IncorrectExact" />
+
+    <exclude name="PEAR.Functions.FunctionCallSignature.Indent" />
+
+    <exclude name="Squiz.Commenting.FileComment.SpacingAfterComment" />
+    <exclude name="Squiz.Commenting.FunctionComment.MissingParamTag" />
+    <exclude name="Squiz.Commenting.InlineComment.InvalidEndChar" />
```

Comment:
Are we excluding these 2 inline comment sniffs intentionally? I would prefer to enforce these as this is how other core inline comments are done: https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#5-inline-comments

Reply:
I think that `Squiz.Commenting.InlineComment.InvalidEndChar` sniff will be invoked if anything except `.`, `!` or `?` is used. I looked at what `twentynineteen` used and copied that over. Not 100% why this was added in the first place 🤔

Reply:
hmmmm :thinking: yeah seems like it just slipped in unexpectedly there. Other comments look to have suggestions here anyways making my point moot. We can reevaluate later about it if needed :)